tests/acceptance/features/bootstrap/FavoritesContext.php

```diff
@@ -71,8 +70,7 @@ public function userFavoritesElementUsingWebDavApi(string $user, string $path):v
 	 * @return void
 	 */
 	public function userHasFavoritedElementUsingWebDavApi(string $user, string $path):void {
-		$this->userFavoritesElement($user, $path);
-		$this->featureContext->theHTTPStatusCodeShouldBeSuccess();
+		$this->featureContext->theHTTPStatusCodeShouldBeBetween(200, 209, $this->userFavoritesElement($user, $path));
```

Comment:
I think favourite should give multistatus so `207` should work here please check
```suggestion
		$this->featureContext->theHTTPStatusCodeShouldBe(207, $this->userFavoritesElement($user, $path));
```